repository/spark/operator/operator.yaml

```diff
@@ -7,22 +7,34 @@ appVersion: "2.4.3"
 maintainers:
   - name: Anton Kirillov
     email: akirillov@d2iq.com
-  - name: Uladzimir Krautsou
-    email: ukrautsou.c@d2iq.com
+  - name: Alexander Lembiewski
+    email: alembiyeuski.c@d2iq.com
 url: https://spark.apache.org/
 tasks:
-  - name: deploy
+  - name: deploy-cluster-scoped-resources
     kind: Apply
     spec:
       resources:
         - spark-operator-crds.yaml
         - spark-operator-rbac.yaml
+  - name: deploy-sa
+    kind: Apply
+    spec:
+      resources:
         - spark-operator-serviceaccount.yaml
         - spark-rbac.yaml
         - spark-serviceaccount.yaml
+  - name: deploy-webhook
+    kind: Apply
+    spec:
+      resources:
         - webhook-init-job.yaml
-        - spark-operator-deployment.yaml
         - webhook-service.yaml
```

Comment:
IIRC, we had some issues with webhook service dependency on init-job for certs generation. Shall the init job deployment happen-before the service deployment?

Reply:
Yes, definitely. Move webhook installation into a separate step.